features/proposed/FxA-99-improved-login-notification/README.md

```diff
@@ -0,0 +1,115 @@
+
+# Fxa-89: Improved Login Notifications
+
+## Problem Summary
+
+The current emails we use to notify users of logins
+contain only minimal information about the event,
+and present it in a potentially-confusing format.
+We can increase the usefulness of these emails
+by providing more detail
+and presenting it in a more user-friendly format.
+
+### Assumptions
+
+* Users expect to know what sort of device
+  was being used to log in, and its approximate location.
+
+* IP-based geoprofiling
```

Comment:
What are you thoughts on expanding this assumption a little? The location data should have a corresponding confidence value associated with it. What if the location is only shown if it breaks a certain confidence level, otherwise it isn't shown? Or maybe wording in the email a little different if level is low?